### ISSUE TITLE
selectors with wrapped schemas

### ISSUE THREAD

**Issue by ikitommi** [metosin/schema-tools#20]
```clj
(st/get-in {:a (s/maybe {:schema s/Bool})} [:a :schema])
;  {:schema java.lang.Boolean}
```

is bit surprising as the `:schema` matches Maybes internals. Should return `s/Bool`...?

**Comment by Deraen** [comment]
Partially implemented by b9f8f069c486ccdf42f428e0a34f88437a78d128

**Comment by Deraen** [comment]
This was tried but decided it was even less consistent to automatically walk over Schema internal fields. I guess we should just close this as I don't think there is any solution to this.
